src/App.jsx

```diff
@@ -31,6 +33,7 @@ const App = () => {
           <Route path = "/Letter" component = { OpenLetter } />
           <Route path = "/Contact" component = { ContactUs } />
           <Route path = "/Analysis" component = { Analysis } />
+          <Route path = "/Testimonial" component = { Testimonial } />
```

Comment:
We're starting to have a lot of code that's being copy+pasted throughout, so we should start looking into creating more reusable components in the future, and abstracting some of the shared logic.

Reply:
New issue to track this in #40